### ISSUE TITLE
certbot not installable on ubuntu 20.4 causing threebot to fail starting

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/js-sdk#101]
I'm using ubuntu 20.04

```
sudo add-apt-repository ppa:certbot/certbot;
[sudo] password for xmonader: 
 This is the PPA for packages prepared by Debian Let's Encrypt Team and backported for Ubuntu.

Note: Packages are only provided for currently supported Ubuntu releases.
 More info: https://launchpad.net/~certbot/+archive/ubuntu/certbot
Press [ENTER] to continue or Ctrl-c to cancel adding it.

Hit:1 http://archive.canonical.com/ubuntu focal InRelease
Hit:2 http://eg.archive.ubuntu.com/ubuntu focal InRelease           
Ign:3 http://ppa.launchpad.net/certbot/certbot/ubuntu focal InRelease
Get:4 http://security.ubuntu.com/ubuntu focal-security InRelease [107 kB]
Get:5 http://eg.archive.ubuntu.com/ubuntu focal-updates InRelease [107 kB]                             
Err:6 http://ppa.launchpad.net/certbot/certbot/ubuntu focal Release                                                  
  404  Not Found [IP: 91.189.95.83 80]
Get:7 http://eg.archive.ubuntu.com/ubuntu focal-backports InRelease [98.3 kB]                                        
Get:8 http://eg.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [204 kB]
Get:9 http://security.ubuntu.com/ubuntu focal-security/main amd64 DEP-11 Metadata [21.2 kB]
Get:10 http://eg.archive.ubuntu.com/ubuntu focal-updates/main amd64 DEP-11 Metadata [105 kB]
Get:11 http://eg.archive.ubuntu.com/ubuntu focal-updates/universe amd64 DEP-11 Metadata [152 kB]
Get:12 http://security.ubuntu.com/ubuntu focal-security/universe amd64 DEP-11 Metadata [31.5 kB] 
Get:13 http://eg.archive.ubuntu.com/ubuntu focal-backports/universe amd64 DEP-11 Metadata [532 B]                    
Reading package lists... Done              
E: The repository 'http://ppa.launchpad.net/certbot/certbot/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.


```

```
➜  js-sdk git:(development) ✗  sh /home/xmonader/sandbox/var/cmds/threebot_default.sh                                                         
2020-06-24 21:06:08.769 | ERROR    | jumpscale.tools.errorhandler.errorhandler:_handle_exception:55 - jumpscale.core.exceptions.exceptions.Not
Found: python-certbot-nginx is not installed.                                                                                                 
Visit https://github.com/threefoldtech/js-sdk/blob/development/docs/wiki/quick_start.md for installation guide                                
Traceback (most recent call last):                                                                                                            
> File "/home/xmonader/wspace/js/js-sdk/.venv/bin/jsng", line 11, in <module>                                                                 
    load_entry_point('js-ng', 'console_scripts', 'jsng')()              
    └ <function load_entry_point at 0x7f5afd2e7820>                     

  File "/home/xmonader/wspace/js/js-sdk/.venv/src/js-ng/jumpscale/entry_points/jsng.py", line 24, in run
    sys.exit(print(eval(sys.argv[1])))                                  
    │   │               │   └ ['/home/xmonader/wspace/js/js-sdk/.venv/bin/jsng', 'j.servers.threebot.start_default(wait=True)']
    │   │               └ <module 'sys' (built-in)>                     
    │   └ <built-in function exit>                                      
    └ <module 'sys' (built-in)>                                         

  File "<string>", line 1, in <module>                                  

  File "/home/xmonader/wspace/js/js-sdk/jumpscale/servers/threebot/__init__.py", line 14, in start_default
    server.start(wait=wait)                                             
    │      │     │    └ True                                            
    │      │     └ True                                                 
    │      └ <function ThreebotServer.start at 0x7f5afb996d30>          
    └ ThreebotServer(_Base__instance_name='default', _Base__parent=None, ___package_manager=<jumpscale.core.base.factory.jumpscales...

  File "/home/xmonader/wspace/js/js-sdk/jumpscale/servers/threebot/threebot.py", line 514, in start
    self.check_dependencies()                                           
    │    └ <function ThreebotServer.check_dependencies at 0x7f5afb996ca0>
    └ ThreebotServer(_Base__instance_name='default', _Base__parent=None, ___package_manager=<jumpscale.core.base.factory.jumpscales...

  File "/home/xmonader/wspace/js/js-sdk/jumpscale/servers/threebot/threebot.py", line 491, in check_dependencies
    raise j.exceptions.NotFound(f"python-certbot-nginx is not installed.\n{install_msg}")
          │ └ <property object at 0x7f5afc15cef0>                       
          └ <jumpscale.god.J object at 0x7f5afc8ae4f0>                  
  File "/home/xmonader/wspace/js/js-sdk/jumpscale/servers/threebot/threebot.py", line 491, in check_dependencies
    raise j.exceptions.NotFound(f"python-certbot-nginx is not installed.\n{install_msg}")
          │ └ <property object at 0x7f5afc15cef0>                       
          └ <jumpscale.god.J object at 0x7f5afc8ae4f0>                  

jumpscale.core.exceptions.exceptions.NotFound: python-certbot-nginx is not installed.
Visit https://github.com/threefoldtech/js-sdk/blob/development/docs/wiki/quick_start.md for installation guide
```

**Comment by xmonader** [comment]
`pip3 install certbot-nginx` worked :)

**Comment by grimpy** [comment]
we could just add it to the poetry depdendencies instead of relying on the host to have it isntalled via apt or whatever packagemanager

**Comment by xmonader** [comment]
fixed in c07ef50db0895cad1854ea7a0094a94024754a14

**Comment by Pishoy** [comment]
verified , certbot installed by `poetry update`

```
root@803edbf8fb80:/js-sdk/js-sdk# poetry update

  - Installing certbot-nginx (1.5.0)


(js-sdk-c4ChsqJN-py3.8) root@803edbf8fb80:/js-sdk/js-sdk# which certbot

/root/.cache/pypoetry/virtualenvs/js-sdk-c4ChsqJN-py3.8/bin/certbot

```
js-sdk version

```
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
(js-sdk-c4ChsqJN-py3.8) root@803edbf8fb80:/js-sdk/js-sdk# git log -1
commit 1e5acd0c8e3ea01a7a882daf839a9171f0ade50d (HEAD -> development, origin/development, origin/HEAD)
Author: AhmedSa-mir <asamir97@hotmail.com>
Date:   Thu Jun 25 13:11:37 2020 +0200

    Renaming js files names to stick to convention
```
